### ISSUE TITLE
Feature Request: Add more logs

### ISSUE THREAD

**Issue by iliarostovtsev** [virtualmin/virtualmin-gpl#201]
Hi,

I was thinking why don't we utilize `journalctl` command that is widely available nowadays, practically on all distros that we support?

I would like us to use it to pipe `journalctl` command output, using current `syslog/save_log.cgi` module and add the following log under **Virtualmin > Logs and Reports** section, based on virtual server:

 **User Log** -  `su - virtual_server_username -c "journalctl --user"`

For **Webmin > System > System Logs** to add more unit based logs, like:

 1. Boot log - `journalctl -b`
 2. BIND log - `journalctl -u named`
 3. Apache log - `journalctl -u apache2`
 4. MySQL log - `journalctl -u mysql`

.. and etc

Does it worth a try?

**Comment by iliarostovtsev** [comment]
It would also be interesting to implement live log with `journalctl -f`.

**Comment by iliarostovtsev** [comment]
> It would also be interesting to implement live log with journalctl -f.

Oh, right, that wouldn't work as expected. We can already use `journalctl -n 1000` with 5 seconds refresh.

**Comment by jcameron** [comment]
Interesting idea - so do you mean adding the output of `journalctl` as an option to view, similar to how it already supports `dmesg` ?

**Comment by jcameron** [comment]
Actually I already added that, although it shows all service's logs.

**Comment by iliarostovtsev** [comment]
Yes, we do have `journalctl -n 1000` but I was thinking that, perhaps splitting it to display logs for different units separately, and adding _User Log_ under **Virtualmin > Logs and Reports** would be an improvement.

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
